packages/manager/modules/pci/src/projects/project/failover-ips/failover-ips.routing.js

```diff
@@ -59,6 +59,9 @@ export default /* @ngInject */ ($stateProvider) => {
             })),
           ),
 
+      failoverIpsRegions: /* @ngInject */ (failoverIps) =>
+        Array.from(new Set(failoverIps.map(({ geoloc }) => geoloc))),
```

Comment:
suggestion: Can we use `geoLocation` instead of `geoLoc`?

Reply:
This value is coming from API side.
My goals here is just to have a list of regions for all failover services.
`geoloc` is same things as `region` field, but API name it like this.

Let me know if you need more info @anooparveti.